client/src/Components/Pages/Questionnaire/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PersonalDetails1 from './PersonalDetails1/index';
+import BasicDetails from './BasicDetails/index';
+
+
+import './style.css'
+
+class Questionnaier extends Component {
+    
+
+    render() { 
+        return (
+             <div>
+               <BasicDetails />
+               <PersonalDetails1 />
```

Comment:
In the future, this class can use state to keep track of which "screen" should be shown, and could then show only the appropriate one at any given time.  And of course have the next button to change that screen :)

Reply:
that is right. actually, I am working on switch component I will push it after finish edits.